<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveWatch</class>
 <widget class="QWidget" name="LiveWatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_roomName">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>xxx的直播间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ownerName">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_gift">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>礼物: 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>关注: 9009</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>积分: 876</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>举报</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_video">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>240</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;直播区域&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_watch">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>观看直播</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_sound">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>打开声音</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_quit">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>退出直播间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_gift">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>打赏</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>在线用户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="list_onlineUserName">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>130</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>130</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>发言区</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="list_danmu">
           <property name="minimumSize">
            <size>
             <width>220</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>220</width>
             <height>200</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="line_danmuSend"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_danmuSend">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>发送弹幕</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
